packages/treasury/src/vaultManager.js

```diff
@@ -160,9 +159,10 @@ export function makeVaultManager(
     const quote = await E(outstandingQuote).getPromise();
     // When we receive a quote, we liquidate all the vaults that don't have
     // sufficient collateral, (even if the trigger was set for a different
-    // level) because we use the actual price ratio plus margin here.
+    // level) because we use the actual price ratio plus margin here. Use
+    // ceilDivide to round up because ratios above this will be liquidated.
     const quoteRatioPlusMargin = makeRatioFromAmounts(
-      divideBy(getAmountOut(quote), liquidationMargin),
+      ceilDivideBy(getAmountOut(quote), liquidationMargin),
```

Comment:
I tried a little mutation testing (changed this to `floorDivideBy`): our tests are not sensitive to this change.

That seems relatively important, if not critical.